docs/tokens/nft-tokens.md

```diff
@@ -264,14 +264,12 @@ For NFT:
 - ESDTRoleNFTBurn : this role allows one to burn quantity of a specific NFT
 - ESDTRoleNFTUpdateAttributes : this role allows one to change the attributes of a specific NFT
 - ESDTRoleNFTAddURI : this role allows one add URIs for a specific NFT
-- ESDTTransferRole : this role enables transfer only to specified addresses. The owner of the NFT and the address with the ESDTTransferRole should be located on the same shard. The addresses with the transfer role can transfer anywhere.
 
 For SFT:
 
 - ESDTRoleNFTCreate : this role allows one to create a new SFT
 - ESDTRoleNFTBurn : this role allows one to burn quantity of a specific SFT
 - ESDTRoleNFTAddQuantity : this role allows one to add quantity of a specific SFT
-- ESDTTransferRole : this role enables transfer only to specified addresses. The owner of the SFT and the address with the ESDTTransferRole should be located on the same shard. The addresses with the transfer role can transfer anywhere.
```

Comment:
same here

Reply:
We need to leave this in, but only remove this part:
"The owner of the NFT and the address with the ESDTTransferRole should be located on the same shard. "

Reply:
only removed this part:
"The owner of the NFT and the address with the ESDTTransferRole should be located on the same shard. "